src/lib/m2/index.js

```diff
@@ -157,7 +157,11 @@ export default new r.Struct({
 
     this.transparencies.forEach((transparency) => {
       if (transparency.animated) {
-        animated = true;
+        if (transparency.keyframeCount > 1) {
+          animated = true;
+        } else if (transparency.firstKeyframe.value !== 32767) {
+          animated = true;
```

Comment:
They are fixed precision 16-bit floats, I believe. You divide by 32767 to get the number. Any pointers on how to set that up with an animation block?